src/main/java/uk/gov/justice/digital/hmpps/keyworker/rolemigration/RoleAssignmentsService.java

```diff
@@ -1,28 +1,34 @@
 package uk.gov.justice.digital.hmpps.keyworker.rolemigration;
 
+import com.microsoft.applicationinsights.TelemetryClient;
 import lombok.extern.slf4j.Slf4j;
 import lombok.val;
 import org.springframework.security.access.prepost.PreAuthorize;
 import org.springframework.stereotype.Service;
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.client.HttpClientErrorException;
+import uk.gov.justice.digital.hmpps.keyworker.dto.RoleAssignmentStats;
 import uk.gov.justice.digital.hmpps.keyworker.dto.RoleAssignmentsSpecification;
 
 import javax.validation.Valid;
-import java.util.List;
-import java.util.Set;
-import java.util.TreeSet;
-import java.util.stream.Collectors;
+import java.util.*;
+
+import static java.util.stream.Collectors.toCollection;
+import static uk.gov.justice.digital.hmpps.keyworker.rolemigration.RoleAssignmentsService.Status.*;
 
 @Service
 @Slf4j
 @Validated
 public class RoleAssignmentsService {
 
+    public enum Status {SUCCESS, FAIL, IGNORE}
```

Comment:
this enum seems to be unnecessary.  It is contained within this class and therefore just adds complexity, especially when we then have to cope with the enum not being set.  Couldn't there just be individual incrementors on the stats class instead?

Reply:
we have moved the enum out of the class but are still using it

Reply:
enum now removed